images/statefun-workshop.svg

```diff
@@ -1,320 +1,378 @@
+         preserveAspectRatio="none"
+         height="1"
+         width="1" /></mask></defs><sodipodi:namedview
+     inkscape:current-layer="g18"
+     inkscape:window-maximized="0"
+     inkscape:window-y="0"
+     inkscape:window-x="0"
+     inkscape:cy="240.36728"
+     inkscape:cx="503.5614"
+     inkscape:zoom="0.75122549"
+     showgrid="false"
+     id="namedview12"
+     inkscape:window-height="900"
+     inkscape:window-width="1440"
+     inkscape:pageshadow="2"
+     inkscape:pageopacity="0"
+     guidetolerance="10"
+     gridtolerance="10"
+     objecttolerance="10"
+     borderopacity="1"
+     inkscape:document-rotation="0"
+     bordercolor="#666666"
+     pagecolor="#ffffff" /><g
+     transform="matrix(1.3333333,0,0,-1.3333333,-24.438595,648.36728)"
+     inkscape:label="fraud-detector"
+     inkscape:groupmode="layer"
+     id="g18"><g
+       id="g20" /><path
+       inkscape:connector-curvature="0"
+       id="path76"
+       style="fill:#6ea8b3;fill-opacity:1;fill-rule:nonzero;stroke:none"
+       d="m 268.0807,261.8344 c 0,31.6281 25.647,57.2677 57.2842,57.2677 31.6372,0 57.2843,-25.6396 57.2843,-57.2677 0,-31.6281 -25.6471,-57.2677 -57.2843,-57.2677 -31.6372,0 -57.2842,25.6396 -57.2842,57.2677 z" /><path
+       d="m 268.0807,261.83438 c 0,31.62809 25.64703,57.26772 57.28426,57.26772 31.63723,0 57.28425,-25.63963 57.28425,-57.26772 0,-31.6281 -25.64702,-57.26772 -57.28425,-57.26772 -31.63723,0 -57.28426,25.63962 -57.28426,57.26772 z"
+       style="fill:none;stroke:#004957;stroke-width:1.60394;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:10;stroke-dasharray:none;stroke-opacity:1"
+       id="path80"
+       inkscape:connector-curvature="0" /><text
+       y="-263.59802"
+       x="287.62537"
+       transform="scale(1.0001444,-0.99985561)"
+       style="font-variant:normal;font-weight:normal;font-size:13.998px;font-family:Arial;-inkscape-font-specification:ArialMT;writing-mode:lr-tb;fill:#ffffff;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.239965"
```

Comment:
Forgot one thing. We should try to stick to the same font throughout our material. I believe David settled on Open Sans for the training.